cmd/agent/subcommands/run/internal/settings/runtime_settings_multi_region_failover.go

```diff
@@ -13,42 +13,42 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config/settings"
 )
 
-// HighAvailabilityRuntimeSetting wraps operations to change the High Availability settings at runtime.
-type HighAvailabilityRuntimeSetting struct {
+// MultiRegionFailoverRuntimeSetting wraps operations to change the High Availability settings at runtime.
```

Comment:
Mention of `High Availability` in the comment here

Reply:
Good catch; fixed in 7dad163f1c.